services/GoogleAdSense.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Google AdSense",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.google.com/adsense/tc/2018/UnitedKingdom.html",
```

Comment:
Why this url and not this one? https://www.google.com/adsense/new/localized-terms?gsessionid=sCco6eXpRU7UA1V1qemcHUPldG-ggJZCHpsm76Y19tI

Reply:
They are the same document : this url is the static adress for English speaking european countries, since I could'nt retrieve the document from the second url.

Reply:
The UK being out from the EU, I'm not sure it is very safe to rely on the UK version 😕

Reply:
I agree, yet when searching for documents of English-speaking countries such as Ireland or even Iceland, I always came across this specific document which seems to be not restrictive to UK.